### PR TITLE
Release/v1.4

### PR DESCRIPTION
**Pull request by chunfantasy**
This PR is to merge release branch to master after release.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -10,23 +18,23 @@
 
 ## Version 1.2.3
 ### Bugfixes
-- Fix cropped most recently used files dropdown #142
+* Fixed cropped most recently used files dropdown #142
 
 ## Version 1.2.2
 ### Bugfixes
-- Fix for UICR handling that caused double reset failure #133
+* Fixed for UICR handling that caused double reset failure #133
```
Comment (bihanssen) [inline Changelog.md]:
```suggestion
* Fixed UICR handling that caused double reset failure #133
```

File: Changelog.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+* Added support for nRF52820
+* Added support for modem UART DFU
+
+## Bugfixes
+* Fixed end address for regions displayed
+
 ## Version 1.3.1
 ## Bugfixes
-- Fix programming nRF51 devices with UICR #156
+* Fixed programming nRF51 devices with UICR #156
```
Comment (bihanssen) [inline Changelog.md]:
```suggestion
* Fixed programming of nRF51 devices with UICR #156
```

File: Changelog.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
 
 ## Version 1.3.0
 ### New features
```
Comment (bihanssen) [inline Changelog.md]: To continue unifying phrases: s/Supported/Added support for/

File: Changelog.md [hunk 4]
```diff
@@ -10,23 +18,23 @@
 
 ## Version 1.2.3
 ### Bugfixes
-- Fix cropped most recently used files dropdown #142
+* Fixed cropped most recently used files dropdown #142
```
Comment (bihanssen) [inline Changelog.md]:
```suggestion
* Fixed cropping of most recently used files dropdown #142
```

File: Changelog.md [hunk 5]
```diff
@@ -10,23 +18,23 @@
...
-- Added modem DFU support
-- Added list of devices and details of current device to system report
+* Added modem DFU support
+* Added list of devices and details of current device to system report
 ### Updates
-- Updated algorithm of detecting application regions
-- Added nRF52810 to supported devices
-- Added SdReq for SoftDevice S140 v6.1.0
+* Updated algorithm of detecting application regions
+* Added nRF52810 to supported devices
```
Comment (bihanssen) [inline Changelog.md]:
```suggestion
* Added support for nRF52810
```

File: Changelog.md (hunk 3 above)
Comment (chunfantasy) [reply Changelog.md]: Good point!

**Pull request merged**